### PR TITLE
Introduce Degree model

### LINKED ISSUE #267: Allow a course to have multiple degrees
Allow a course to have multiple degrees (e.g. Bachelor AND Master).
This should be done by checkboxes instead of the current dropdown menu in the course form.

### PR DESCRIPTION
**Pull request by janno42**
Fix #267
Allows a course to have multiple degrees. Degrees are now also translatable and sortable.

### CODE REVIEW

**Comment by janno42** [comment]
reminder: merge conflicts because of #580

**Comment by karyon** [comment]
that's it i guess...

**Comment by janno42** [comment]
i will do the prefetch stuff and the rebase some other day.

File: evap/evaluation/models.py [hunk 1]
```diff
@@ -147,7 +161,7 @@ class Course(models.Model, metaclass=LocalizeModelBase):
     type = models.CharField(max_length=1024, verbose_name=_("type"))
 
     # bachelor, master, d-school course
```
Comment (karyon) [inline evap/evaluation/models.py]: could you change this to something neutral like "e.g. Bachelor Information Science, Master something else"

Comment (janno42) [inline evap/evaluation/models.py]: i will remove it. the terminology clearly states what a degree is.

Comment (janno42) [inline evap/evaluation/models.py]: or well, i'll just remove the d-school stuff. that should be okay.

**Comment by karyon** [comment]
:ship:

**Pull request merged**